parts/k8s/kubernetescustomscript.sh

```diff
@@ -41,12 +41,13 @@ function holdWALinuxAgent() {
     fi
 }
 
+testOutboundConnection
```

Comment:
Until we prove that we can bootstrap a k8s control plane w/ no outbound internet access we should put this back.